### PR TITLE
Added IRELAND Flag page to existing FRANCE page

### LINKED ISSUE #10: Add IRELAND Flag

### PR DESCRIPTION
**Pull request by harshpatel2212**
closes #10

### CODE REVIEW

**Comment by GPrince00** [comment]
@harshpatel2212  You shouldn't create a new folder as you have done. You should just add the file of the flag as the other flags .

**Comment by harshpatel2212** [comment]
@GPrince00 okay, I am adding the source images in the assets, and France file

**Comment by harshpatel2212** [comment]
@GPrince00 buddy, is it okay now?

**Comment by harshpatel2212** [comment]
Can you please verify now and do the needful!

File: pages/ireland.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="shortcut icon" href="../assets/irelandFavicon.png">
+    <title>France</title>
```
Comment (GPrince00) [inline pages/ireland.html:10]: Can you change the title to Ireland

**Review by GPrince00 — CHANGES_REQUESTED** [review]

File: pages/ireland.html (hunk 1 above)
Comment (GPrince00) [reply pages/ireland.html:10]: @harshpatel2212 just this, I can resolve the conflict for you

Comment (harshpatel2212) [reply pages/ireland.html:10]: ya sure

**Review by GPrince00 — APPROVED** [review]

**Review by GPrince00 — APPROVED** [review]

**Pull request merged**